packages/utils/src/index.js

```diff
@@ -266,3 +266,31 @@ export function getWordCount(editorState: EditorState): number {
   const wordArray = cleanString.match(/\S+/g) // matches words according to whitespace
   return wordArray ? wordArray.length : 0
 }
+
+export function mergeEntityData(
+  editorState: EditorState,
+  entityKey: string,
+  data: Object
+): EditorState {
+  const newContentState = editorState
+    .getCurrentContent()
+    .mergeEntityData(entityKey, data)
+
+  return EditorState.push(editorState, newContentState, 'apply-entity')
```

Comment:
Not sure if this actually adds any value, would you mind moving that into the example?

Reply:
Due to nothing relate to it(till this moment), i just remove it.